freqtrade/commands/cli_options.py

```diff
@@ -398,6 +398,30 @@ def __init__(self, *args, **kwargs):
         help='Select only best epochs.',
         action='store_true',
     ),
+    "hyperopt_list_min_avg_time": Arg(
+        '--min-avg-time',
+        help='Select epochs on above average time.',
+        type=check_int_nonzero,
+        metavar='INT',
```

Comment:
this should better be float... 
it will require introducing check_float_positive() checking function, I'll do this later on top of this PR...